README.md

```diff
@@ -58,6 +58,13 @@ If you'd like to make modifications to the codebase, run the dev server with: `n
 
 If you'd like to rebuild the `/dist` folder with your codebase changes, run `npm run build`.
 
+
+##### Integrate Tests
+
+You will need JDK of version 7 or higher as instructed here
+http://nightwatchjs.org/gettingstarted#selenium-server-setup
```

Comment:
1) Can we reasonably expect a system to have JDK 7+ installed?

2) If we want these integration tests to automatically run in CI (do we?), we need to make sure we're provisioning Travis correctly. @webron?